linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_mac_modification_usr_share/oval/shared.xml

```diff
@@ -4,6 +4,14 @@
       mandatory access controls (SELinux) in usr/share/selinux are enabled.") }}}
 
     <criteria operator="OR">
+{{%- if 'sle15' in product -%}}
```

Comment:
This condition also matches any other string that contains "sle15" as a substring, eg. "sle158", "ssle15". That means it can match unwanted products in future. Normally, you should do `product == "sle15"`, or better, `product in ["sle15"]`. The latter will allow people do have an easier expansion in future.

Reply:
Thanks for the tip changed it in [c3a8fb7](https://github.com/ComplianceAsCode/content/pull/10883/commits/c3a8fb73e4cfcd626b9d2f19f7247585fdc9efb7)